python/cudf/cudf/_libxx/lib.pyx

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cudf._libxx.lib cimport *
+from enum import Enum
+
+
+class InterpolationEnum(Enum):
```

Comment:
Do we want the `Enum` suffix for these? I'm not opposed - but if we're doing that, we should probably change `MaskState` to `MaskStateEnum` as well.

Reply:
In generally, I don't like suffuxes, but in this case the names map 1-to-1 with libcudf, so we have less control over name collisions. I don't know if this is a problem. Does anyone feel strongly about this? I'm happy to change it (I'm on the fence myself).

Reply:
+1 on not having the suffixes, if name collisions are a problem they can be handled by import aliasing or accessing it via the module.